config/install/easy_breadcrumb.settings.yml

```diff
@@ -2,7 +2,7 @@ applies_admin_routes: true
 include_invalid_paths: true
 excluded_paths: ''
 replaced_titles: ''
-custom_paths: "/admin/school :: Schools\r\n/admin/school/overview :: Schools\r\n/admin/school/pending :: Schools | /admin/school/overview  :: Pending-Schools\r\n/admin/school/mapping :: Schools | /admin/school :: School-Mapping\r\n/admin/student :: Students\r\n/admin/content/mini_node/add/academic_session :: Academic Sessions List|/academic-sessions :: Academic Session\r\n/admin/structure/taxonomy/manage/school/add :: Add School\r\n/admin/structure/taxonomy/manage/school/overview :: School List\r\n/admin/config/regional/translate :: User Interface Translation\r\n/admin/people :: People\r\n/admin/people/create :: People | /admin/people :: Add User\r\n/admin/location/bulk-upload :: Location :: Bulk Upload\r\n/admin/structure/taxonomy/manage/location/overview :: Locations\r\n/user/register :: School Register\r\nregex!/school/registration/([^/]*)/([^/]*)/edit :: Edit School Registration\r\nregex!/school/registration/([^/]*)/([^/]*)/view :: School Registration\r\n/student/application :: Student-Application\r\n/admin/content/mini_node/add/student_details ::  Add Student Details\r\n/student/login :: Student-Login\r\nregex!/user/([^/]*)/edit :: Edit\r\n/student/application-status :: Student Application Status"
+custom_paths: "/admin/school :: Schools\r\n/admin/school/overview :: Schools\r\n/admin/school/pending :: Schools | /admin/school/overview  :: Pending-Schools\r\n/admin/school/mapping :: Schools | /admin/school :: School-Mapping\r\n/admin/student :: Students\r\n/admin/content/mini_node/add/academic_session :: Academic Sessions List|/academic-sessions :: Academic Session\r\n/admin/structure/taxonomy/manage/school/add :: Add School\r\n/admin/structure/taxonomy/manage/school/overview :: School List\r\n/admin/config/regional/translate :: User Interface Translation\r\n/admin/people :: People\r\n/admin/people/create :: People | /admin/people :: Add User\r\n/admin/location/bulk-upload :: Location :: Bulk Upload\r\n/admin/structure/taxonomy/manage/location/overview :: Location List\r\n/user/register :: School Register\r\nregex!/school/registration/([^/]*)/([^/]*)/edit :: Edit School Registration\r\nregex!/school/registration/([^/]*)/([^/]*)/view :: School Registration\r\n/student/application :: Student-Application\r\n/admin/content/mini_node/add/student_details ::  Add Student Details\r\n/student/login :: Student-Login\r\nregex!/user/([^/]*)/edit :: Edit\r\n/student/application-status :: Student Application Status\r\n/admin/structure/taxonomy/manage/location/add :: Location List | /admin/structure/taxonomy/manage/location/overview :: Add Location"
```

Comment:
Didn't get this
admin/structure/taxonomy/manage/location/add :: Location List 
admin/structure/taxonomy/manage/location/overview :: Add Location

Don't you think it should be opposite?

Reply:
We are showing the Location list tot he state admin, from that we have the option to add location.
So the breadcrumb should be like HOME >> Location List >> Add Location

Reply:
`admin/structure/taxonomy/manage/location/add` This url lands on the location add page, right?

Reply:
yes,
/admin/structure/taxonomy/manage/location/add :: Location List | /admin/structure/taxonomy/manage/location/overview :: Add Location". Please check the syntax here, | denotes the Breadcrumb Link Name(separated by the destination link).

Reply:
Share the SS

Reply:
![image](https://github.com/Indus-Action-Initiatives/rte-mis/assets/124239255/0dd759f4-4401-447a-874d-b5423315bb62)
![image](https://github.com/Indus-Action-Initiatives/rte-mis/assets/124239255/951f8ee3-3601-45c2-9508-39d884344e97)

Reply:
Cool, Thanks. Understood